app/javascript/packs/application.js

```diff
@@ -43,6 +43,7 @@ export { default as Shortcuts } from '../src/javascripts/shortcuts.js';
 export { default as Upload } from '../src/javascripts/uploads.js';
 export { default as Utility } from '../src/javascripts/utility.js';
 export { default as Ugoira } from '../src/javascripts/ugoira.js';
+export { default as Thumbnails } from '../src/javascripts/thumbnails.js';
```

Comment:
I recognize that this was probably just done for consistency, but the default export of thumbnails.js doesn't seem to be used for anything, so this is equivalent to an import statement loading it into window scope (since it has a window method).

Reply:
I'll have to double check, but I believe that those exports are to local scope only, and not published into the window object automatically. It was indeed done for consistency, and I have a feeling it's being done this way to avoid linting rules surrounding import for side effects only.

Reply:
From testing, there are all no-op commands. They both don't push things into the window object, and they don't have any side effects, as they are imported through the importAll command above. I will remove them entirely.

Reply:
I found out what these are for. It appears that this somehow gets exported into the `Danbooru` namespace which is placed into the window object. So these are still required.